codespell_lib/data/dictionary.txt

```diff
@@ -13686,13 +13773,20 @@ contiguious->contiguous
 contiguities->continuities
 contiguos->contiguous
 contiguous-non->non-contiguous
+contina->contain
```

Comment:
I guess `continua` and `fontina` cheese are rare, aren't they?

Reply:
These words are too rare.